docs/spark/tutorials/get-started-ubuntu-instructions.md

```diff
@@ -0,0 +1,117 @@
+
+- Download and install the following: **[.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)** | **[OpenJDK 8](https://openjdk.java.net/install/)** | **[Apache Spark 2.4.1](https://archive.apache.org/dist/spark/spark-2.4.1/spark-2.4.1-bin-hadoop2.7.tgz)**
+- Download and install **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release:
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (e.g., `~/bin/Microsoft.Spark.Worker`).
+  - **IMPORTANT** Create a [new environment variable](https://help.ubuntu.com/community/EnvironmentVariables) `DOTNET_WORKER_DIR` and set it to the directory where you downloaded and extracted the Microsoft.Spark.Worker (e.g., `~/bin/Microsoft.Spark.Worker`).
+
+For detailed instructions, you can see [Building .NET for Apache Spark from Source on Ubuntu](../building/ubuntu-instructions.md).
+
+## Authoring a .NET for Apache Spark App
+
+1. Use the .NET Core CLI to create a console application.
+
+   ```dotnetcli
+   dotnet new console -o HelloSpark
+   ```
+
+1. Install the `Microsoft.Spark` NuGet package into the project from the [spark nuget.org feed](https://www.nuget.org/profiles/spark) - see [Ways to install Nuget Package](https://docs.microsoft.com/en-us/nuget/consume-packages/ways-to-install-a-package).
+
+    ```shell
+    cd HelloSpark
+    dotnet add package Microsoft.Spark
+    ```
+
+1. Replace the contents of the *Program.cs* file with the following code:
+
+    ```csharp
+    using Microsoft.Spark.Sql;
+
+    namespace HelloSpark
+    {
+        class Program
+        {
+            static void Main(string[] args)
+            {
+                var spark = SparkSession.Builder().GetOrCreate();
+                DataFrame df = spark.Read().Json("people.json");
+                df.Show();
+            }
+        }
+    }
+    ```
+
+1. Use the .NET Core CLI to build the application:
+
+    ```dotnetcli
+    dotnet build
+    ```
+
+## Running your .NET for Apache Spark App
+
+1. Open your terminal and navigate into your app folder.
+
+    ```shell
+    cd <your-app-output-directory>
+    ```
+
+1. Create *people.json* with the following content:
+
+    ```json
+    {"name":"Michael"}
+    {"name":"Andy", "age":30}
+    {"name":"Justin", "age":19}
+    ```
+
+1. Run your app.
+
+    ```shell
+    spark-submit \
+    --class org.apache.spark.deploy.dotnet.DotnetRunner \
+    --master local \
+    microsoft-spark-2.4.x-<version>.jar \
+    dotnet HelloSpark.dll
+    ```
+
+    > [!NOTE]
+    > This command assumes you have downloaded Apache Spark and added it to your PATH environment variable to be able to use `spark-submit`, otherwise, you would have to use the full path (e.g., `~/spark/bin/spark-submit`). For detailed instructions, you can see [Building .NET for Apache Spark from Source on Ubuntu](../building/ubuntu-instructions.md).
```

Comment:
```suggestion
    > This command assumes you have downloaded Apache Spark and added it to your PATH environment variable to be able to use `spark-submit`, otherwise, you would have to use the full path (for example, *~/spark/bin/spark-submit*). For detailed instructions, you can see [Building .NET for Apache Spark from Source on Ubuntu](../building/ubuntu-instructions.md).
```